great_ape_safe/ape_api/badger.py

```diff
@@ -175,7 +175,7 @@ def claim_bribes_hidden_hands(self, claim_from_strat=True) -> dict:
         aggregate = {"tokens": [], "amounts": []}
         for item in data:
             aggregate["tokens"].append(item["token"])
-            aggregate["amounts"].append(item["claimMetadata"]["amount"])
+            aggregate["amounts"].append(item["claimable"].replace('.', ''))
```

Comment:
By replacing `.` with `''`, we are counting on the amount of decimals provided to be complete and I am not sure it will always be the case. For instance, in the example from the issue, the claimable is `739.356929` but removing the `.` leads to `739356929 wei` which is only: `0.000000000739356929`. Maybe I am missing something?

Reply:
usdc only has 6 decimals

Reply:
Ah, that's what I was missing... 😬